src/ExportHistory/Client/DefaultExportHistoryJobClient.cs

```diff
@@ -0,0 +1,216 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Grpc.Core;
+using Microsoft.DurableTask.Client;
+using Microsoft.DurableTask.Client.Entities;
+using Microsoft.DurableTask.Entities;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.DurableTask.ExportHistory;
+
+/// <summary>
+/// Convenience client for managing export jobs via entity signals and reads.
+/// </summary>
+public sealed class DefaultExportHistoryJobClient(
+    DurableTaskClient durableTaskClient,
+    string jobId,
+    ILogger logger,
+    ExportHistoryStorageOptions storageOptions) : ExportHistoryJobClient(jobId)
+{
+    readonly DurableTaskClient durableTaskClient = Check.NotNull(durableTaskClient, nameof(durableTaskClient));
+    readonly ILogger logger = Check.NotNull(logger, nameof(logger));
+    readonly ExportHistoryStorageOptions storageOptions = Check.NotNull(storageOptions, nameof(storageOptions));
+    readonly EntityInstanceId entityId = new(nameof(ExportJob), jobId);
+
+    /// <inheritdoc/>
+    public override async Task CreateAsync(ExportJobCreationOptions options, CancellationToken cancellation = default)
+    {
+        try
+        {
+            Check.NotNull(options, nameof(options));
+
+            // Determine default prefix based on mode if not already set
+            string? defaultPrefix = $"{options.Mode.ToString().ToLower(System.Globalization.CultureInfo.CurrentCulture)}/";
+
+            // If destination is not provided, construct it from storage options
+            ExportJobCreationOptions optionsWithDestination = options;
+            if (options.Destination == null)
```

Comment:
I feel like this is a better user experience if we don't let them create a job without a destination. constructing a default I think is a recipe for people exporting to locations they didn't mean to.